sycl/test/basic_tests/handler/parallel_for_arg_restrictions.cpp

```diff
@@ -1,10 +1,11 @@
 // RUN: %clangxx -fsycl -fsyntax-only -ferror-limit=0 -Xclang -verify -Xclang -verify-ignore-unexpected=note,warning,error %s
-// RUN: %clangxx -fsycl -fsyntax-only -ferror-limit=0 -Xclang -verify -Xclang -verify-ignore-unexpected=note,warning,error %s -DSYCL2020_CONFORMANT_APIS
+// RUN: %clangxx -fsycl -fpreview-breaking-changes -fsyntax-only -ferror-limit=0 -Xclang -verify -Xclang -verify-ignore-unexpected=note,warning,error %s -DPREVIEW_BREAKING_CHANGES
+// REQUIRES: preview-breaking-changes-lib
```

Comment:
Why `REQUIRES` instead of `%if preview-breaking-changes-lib %{ ... %}`? Also, I don't like the name of the LIT feature. Maybe "preview-breaking-changes-supported" ?